index.html

```diff
@@ -362,11 +362,8 @@ <h1 class="overlay">
 
 var toggleTagVisibility = function () {
   var tagStyleElement = document.getElementById('tagVisibility');
-  if(tagStyleElement.innerHTML === '.tags{display:none;}') {
-    tagStyleElement.innerHTML = '.tags{display:block;}';
-  } else {
-    tagStyleElement.innerHTML = '.tags{display:none;}';
-  }
+  var html = tagStyleElement.innerHTML;
+  tagStyleElement.innerHTML = /^\li div.tags\{display:none;\}$/.test(html) ? 'li div.tags{display:block;}' : 'li div.tags{display:none;}';
```

Comment:
Consider keeping the `if` statement. While it matters less outside a framework, [`if` is faster than ternaries](https://jsperf.com/ternary-operator-vs-if-statement/14).